### PR TITLE
Fix cloudpickle version

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
closes https://github.com/RasaHQ/rasa/issues/5216

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
@tabergma Why is it failing? So which requirements are needed `cloudpickle` and can we maybe increase some of these requirements instead of pinning cloudpickle?

**Comment by tabergma** [comment]
no idea 😅let me try to figure that out

**Comment by tabergma** [comment]
I think it is coming from `tensor2tensor`. It requires `gym`, which requires `cloudpickle`. We cannot increase the version of `tensor2tensor`, so I guess we need to pin `cloudpickle` for now.

**Comment by wochinge** [comment]
Omg, `tensor2tensor` does not pin any dependencies 💥 

> I think it is coming from tensor2tensor. It requires gym, which requires cloudpickle. We cannot increase the version of tensor2tensor. Also the latest version of gym (0.16.0) has requirement cloudpickle~=1.2.0.

What else is requiring `cloudpickle`?

File: setup.py [hunk 1]
```diff
@@ -74,6 +74,8 @@
     "sanic~=19.9.0",
     "sanic-cors==0.9.9.post1",
     "sanic-jwt~=1.3",
+    # needed because of https://github.com/RasaHQ/rasa/issues/5216
+    "cloudpickle==1.2.2",
```
Comment (wochinge) [inline setup.py]:
```suggestion
    "cloudpickle~=1.2.2",
```

**Pull request merged**